HPXMLtoOpenStudio/resources/waterheater.rb

```diff
@@ -976,6 +976,19 @@ def self.apply_hpwh_dxcoil(runner, model, water_heating_system, elevation, obj_n
 
     cop = water_heating_system.additional_properties.cop
 
+    # Adjust COP based on RESNET HERS Addendum 77
+    if not water_heating_system.hpwh_containment_volume.nil?
+      if water_heating_system.hpwh_installed_properly.nil? || (water_heating_system.hpwh_installed_properly == true)
+        runner.registerWarning("HPWHContainmentVolume provided in #{water_heating_system.id} is ignored because it is only used when HPWHInstalledProperly is 'false'.")
+      else
+        if water_heating_system.hpwh_containment_volume < 450.0 && (water_heating_system.backup_heating_capacity == 0.0)
```

Comment:
Can we put a "FUTURE: apply for 120V HPWH and other system types correction may not be accurate for"? Just to track this better outside of that PR.